Podfile

```diff
@@ -34,11 +36,11 @@ target 'SwiftGenKit' do
   end
 end
 
-post_install do | installer |
+post_install do |_installer|
```

Comment:
What's with the underscore here?

Reply:
The parameter is unused. So rubocop says you should either omit the parameter completely, or name it `_` (like in Swift), or name it whatever but with a leading `_` – to give it a name for readability/code understandability purpose, even if unused.

It's just that `bundle exec rubocop -a` (to auto-correct the violations) apparently chose to go with `_installer` (as opposed to just `_` or removing the parameter completely

Reply:
Ah okay that explains it, 👍  for me then.